gnomad/utils/annotations.py

```diff
@@ -1213,10 +1218,11 @@ def generate_pop_json(
     return subpop_record
 
 
-def get_gks(
+def get_gnomad_gks(
```

Comment:
can change this back to get_gks

def get_gks(
    ht: hl.Table,
    variant: str,
    vrs_only: bool = False,
    ancestry_groups: list = None,
    ancestry_group_names: dict = None,
    by_sex: bool = False,
    coverage_ht: hl.Table = None,
) -> dict:
    """
    :param vrs_only: Whether or not to return only the VRS schema for the specified variant. Default is False.
    :param ancestry_group_names: Dictionary containing mapping for ancestry group names, where keys are the shortened names for the ancestry groups, and values are the full label names for that group.
    :param coverage_ht: Table containing coverage annotations, with mean coverage for a locus stored in 'mean' annotation.
    """
    
  then create a gnomad-specific function (this will eventually be put in a different file but need to discuss that)  
  def get_gnomad_gks(version: str, variant:str, by_pops: bool=True, by_sex: bool=True):
    # Based on version, load in proper gnomad ht, coverage ht
    # Warn if by_sex but not ancestry groups (TODO: ask if they ever want by_sex but not by_pops)
    # Call get_gks(fill in params here)